docs/index.md

```diff
@@ -2171,42 +2214,42 @@ let's review its dialogue box options.
 
 This dialogue has the following options:
 
-Root statements ("first level" and "other levels"):
-These let you select which statements (if any) currently
+Root statements/steps ("first level" and "other levels"):
```

Comment:
As for me constantly using statement and step separated by a slash makes reading of this part of the guide (explanation of the bottom-up prover) a bit more complex. I think using just `statement` would be enough.

Reply:
I'll see if I can get rid of the slashes. The tricky part is when the v10 UI says "statement" when "step" is meant, since that's what the user sees. Maybe I can use "statement*" in that case, indicating "statement is what the UI shows, but it means step". Then when v11 arrives I can search for `statement\*`.

Reply:
The thing is, in this case I really do mean *steps* not *statements* - the checkbox in my mind is associated with a step (which contains a statement).

Reply:
Okay, I now say "statement‡" when I really mean *step* but the UI says *statement*. That way the guide uses the same terms as the UI it's describing, but hints at the more correct terminology, and it'll be easier to search-and-replace later.

Reply:
Yes, this is a kind of a compromise between precision and readability. But such things are not avoidable sometimes.